src/main/java/com/minecolonies/blocks/BlockBarrel.java

```diff
@@ -204,17 +205,18 @@ private void doBarrelCompostTick(World world, BlockPos pos, IBlockState state)
         timers.put(pos, timer);
     }
 
+    @NotNull
     @Override
     protected BlockStateContainer createBlockState()
     {
         return new BlockStateContainer(this, BARRELSTATE);
     }
 
-    public boolean onBlockActivated(World worldIn, BlockPos pos, IBlockState state, EntityPlayer playerIn, EnumFacing side, float hitX, float hitY, float hitZ)
+    public boolean onBlockActivated(final World worldIn, final BlockPos pos, final IBlockState state, final EntityPlayer playerIn, final EnumFacing side, final float hitX, final float hitY, final float hitZ)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 207 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)